### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by ahangarha**
## Description

In this pull request these changes are made:

- Add mobile menu design
- Add menu interactive functionality using Javascript
- Install eslint and update hintrc and stylelintrc files

### General requirements

*Check items below by adding x between brackets.*

- [x] There are no linter errors.
- [x] Followed the **Github flow**.
- [x] The documentation is professional.

### HTML/CSS requirements

*Feel free to remove the item(s) not related to this PR.*

- [x] Followed best practices for HTML & CSS
- [x] Followed best practices for Javascript

---
Thank you for taking the time to review this pull request.

### CODE REVIEW

File: assets/js/script.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    mobileMenu.style.display = 'none';
+    document.body.classList.toggle('blury');
+  },
+);
+
+document.querySelectorAll('.mobile-menu-content li a').forEach(
+  (link) => {
+    link.addEventListener('click', () => {
+      mobileMenu.style.display = 'none';
+      document.body.classList.toggle('blury');
```
Comment (andersonlebon) [inline assets/js/script.js]: ## [OPTIONAL]
- [ ] To make your code DRY(don't repeat yourself) I would suggest you put these two lines  in a separate function as these lines are repeated twice.

### Better Implementation
```js
function closeMenu () {
        mobileMenu.style.display = 'none';
        document.body.classList.toggle('blury');
}

function openMenu() {
        mobileMenu.style.display = 'block';
        document.body.classList.toggle('blury');
}

document.querySelector('.menu-toggle').addEventListener('click', openMenu),
document.getElementById('close-mobile-menu-btn').addEventListener( 'click',  closeMenu)

document.querySelectorAll('.mobile-menu-content li a').forEach(
  (link) => {
    link.addEventListener('click', closeMenu )
  }
)
```

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @ahangarha  

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Nice job :clap: using `Javascript` for implementing this mobile menu feature, Hovever one of the requirements says that you should set up both JavaScript and HTML&CSS linters in the project repository.  Kindly follow [these](https://github.com/microverseinc/linters-config/tree/master/html-css-js) intructions for the both linters configuration. Check here 👇  down 

Your configuration | the better configuration with Js
------|-----
image | image



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/js/script.js (hunk 1 above)
Comment (ahangarha) [reply assets/js/script.js]: Thank you @andersonlebon for your suggestion. I just made a toggle function to handle all three scenarios. Now the code looks better and more maintainable.

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi team,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍 
- Project is set up correctly ✔️ 
- Descriptive Pull Request ✔️ 
- No linter errors ✔️ 
- Professional documentation ✔️ 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/js/script.js (hunk 1 above)
Comment (andersonlebon) [reply assets/js/script.js]: You're welcome, sir.

**Pull request merged**